src/internal/message/message.go

```diff
@@ -112,6 +113,13 @@ func Error(err any, message string) {
 	Warnf(message)
 }
 
+func ErrorWebf(err any, w http.ResponseWriter, format string, a ...any) {
```

Comment:
This is great for now but what are your thoughts on adding `statusCode int` as a parameter in the near future?